packages/loader/driver-utils/api-report/driver-utils.api.md

```diff
@@ -30,7 +30,7 @@ import { IStreamResult } from '@fluidframework/driver-definitions';
 import { ISummaryContext } from '@fluidframework/driver-definitions';
 import { ISummaryHandle } from '@fluidframework/protocol-definitions';
 import { ISummaryTree } from '@fluidframework/protocol-definitions';
-import { ITelemetryErrorEvent } from '@fluidframework/core-interfaces';
+import { ITelemetryErrorEventExt } from '@fluidframework/telemetry-utils';
```

Comment:
Presumably we'll be able to rename all of these "Ext" types, now that their corresponding base interfaces are gone? Subsequent PR?

Reply:
Yeah, I wasn't thinking about that step yet but definitely can be done in a subsequent PR.